src/main/resources/css/stylesheet.css

```diff
@@ -1,40 +1,111 @@
 .root{
     -fx-font-size: 12px;
     -fx-font-family: "Courier New";
-    -fx-base: rgb(132, 145, 47);
-    -fx-background: rgb(222,205,205);
-}
+    -fx-alignment: center;
 
+}
 .BorderPane{
     -fx-min-height: "720" ;
     -fx-min-width: "1280";
 }
-/*
-    Styling for containers
- */
-#centerTopPane{
-
+/* Top Pane */
+.LoginAnchor{
+    -fx-min-height: 80px;
+    -fx-max-height: 80px;
+    -fx-font-style: inherit;
 }
-
-#topPane{
-    -fx-alignment: center;
-    -fx-hgap: 10px;
-    -fx-vgap: 10px;
-    -fx-min-height: 50px;
+#login{
+    -fx-min-height: 30px;
+}
+#login:hover{
+    -fx-background-color: cornflowerblue;
+}
+#customerID{
+    -fx-min-height: 30px;
+}
+#customerLabel{
+    -fx-min-height: 30px;
+}
+#showID {
+    -fx-font-size: 16px;
+    -fx-min-height: 30px;
 }
 
+.PickInfo{
```

Comment:
Hvorfor ikke bare sætte height I stedet for alt det min og max? Vi har ikke noget skalerbart, så du burde kunne holde dig til height og width

Reply:
Og er ikke helt sikker på pref height, men virker heller ikke nødvendigt

Reply:
synes også at pref er lidt flydende. Men man kan ikke bare give den height. kun min,max og pref